Keas.Mvc/ClientApp/components/Tags/SearchTags.tsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types";
+import * as React from "react";
+import { Highlighter, Typeahead } from "react-bootstrap-typeahead";
+import 'react-bootstrap-typeahead/css/Typeahead.css';
+
+interface IProps {
+    onSelect?: (tag: string[]) => void;
+    disabled: boolean;
+    defaultValues?: string[];
+    tags: string[];
+}
+
+export default class SearchTags extends React.Component<IProps, {}> {
+
+    public render() {
+        return (
+            <div>
+                <Typeahead
+                    options={this.props.disabled ? [] : this.props.tags}
+                    disabled={this.props.disabled}
+                    multiple={true}
+                    clearButton={true}
+                    onChange={selected => {
+                        this.props.onSelect(selected);
```

Comment:
@laholstege if you are going to call onSelect() here, then it should be a require prop.  Is there a scenario in which we'd want it optional like the IProps interface would suggest?